source/dynamic_data_attachment.md

```diff
@@ -0,0 +1,97 @@
+
+__instance('survey')/root/data[P_Demographic/P_SN = current()/../C_SN]/P_Demographic/P_Sex:__ Will return users the sex (select_one choice) that was entered in P_Sex (parent project) located under the P_Demographic group (parent project) when the value entered in C_SN (child project) matches the value entered in P_SN (parent project). The output of this syntax yields the exact result that a pull data function yields.
+
+__instance('survey')/root/data[P_Demographic/P_SN = current()/../C_SN]/P_Demographic/P_Family:__ Will return users the family size (integer value) that was entered in P_Family (parent project) located under the P_Demographic group (parent project) when the value entered in C_SN (child project) matches the value entered in P_SN (parent project). The output of this syntax yields the exact result that a pull data function yields.
+
+__sum(instance('survey')/root/data/P_Demographic/P_Family):__ Will return users the total sum of what was entered in P_Family (parent project) located under the P_Demographic group (parent project).
+
+__max(instance('survey')/root/data/P_Demographic/P_Family):__ Will return users the maximum value of what was entered in P_Family (parent project) located under the P_Demographic group (parent project).
+
+__min(instance('survey')/root/data/P_Demographic/P_Family):__ Will return users the minimum value of what was entered in P_Family (parent project) located under the P_Demographic group (parent project).
+
+__instance('survey')/root/data[P_Demographic/P_SN = current()/../C_SN]/P_Demographic/P_Language:__ Will return users the language (select_multiple choice) that was entered in P_Language (parent project) located under the P_Demographic group (parent project) when the value entered in C_SN (child project) matches the value entered in P_SN (parent project). The output of this syntax yields the exact result that a pull data function yields.
+
+__instance('survey')/root/data[P_Demographic/P_SN = current()/../C_SN]/P_Demographic/P_Location:__ Will return users the GPS location that was entered in P_Location (parent project) located under the P_Demographic group (parent project) when the value entered in C_SN (child project) matches the value entered in P_SN (parent project). The output of this syntax yields the exact result that a pull data function yields.
+
+## Setting up projects for Dynamic Data Attachment:
+
+A project can be dynamically attached in two different scenarios i.e. when _both the projects i.e. parent project, as well as child project, are owned by the same user_ and when _projects are owned by different users_.
+
+### Both Projects Owned by the Same User:
+
+Once a user has finalized their projects in an xlsform, they will need to do the following to set it up so that they could link them dynamically:
```

Comment:
> Once a user has finalized their projects in an xlsform, they will need to do the following to set it up so that they could link them dynamically:

Maybe rather:

> Follow the steps below to dynamically link projects: